package.json

```diff
@@ -19,10 +19,10 @@
     "ava": "^0.19.0",
     "babel-eslint": "^7.1.1",
     "dustjs-linkedin": "^2.7.5",
-    "eslint": "^3.17.1",
+    "eslint": "^5.15.3",
     "husky": "^0.13.2",
     "memory-fs": "^0.4.1",
-    "webpack": "^2.2.1"
+    "webpack": "^4.29.6"
```

Comment:
Please add `webpack: >=4.0.0` to the `peerDependencies` list

Reply:
The loader still works on 2 and 3. Ok with setting `"webpack": ">=2.0.0"`?

Reply:
peerDependencies pushed up. Thanks!